doc/user_manual/theBibliography.tex

```diff
@@ -277,5 +277,13 @@
         P. L’Ecuyer, ``Efficient and Portable Combined Random Number Generators'',
         Commun. ACM 31, no. 6 (June 1988): 742–751. DOI:10.1145/62959.62969.
     %
+    \bibitem{crystal:1}
+    V.~Previtali, ``Performance Evaluation of a Crystal-enhanced Collimation System for the LHC'',
+    Ph.~D.~thesis, EPFL, Lausanne, Switzerland (2010).
+    %
+    \bibitem{crystal:2}
+    D.~Mirarchi, ``Crystal collimation for LHC'',
+    Ph.~D.~thesis, Imperial College, London, UK (2015).
+    %
```

Comment:
what about adding [this ref](http://cds.cern.ch/record/2694336?ln=en)?

Moreover, does the imported code take into account the [work by F. Forcher](http://cds.cern.ch/search?ln=en&as=1&m1=a&p1=forcher&f1=author&op1=a&m2=a&p2=&f2=title&op2=a&m3=a&p3=&f3=&action_search=Search&c=CERN+Document+Server&sf=&so=a&rm=&rg=100&sc=1&of=hb)?